### PR TITLE
APPS-2714 : Support for presence

### PR DESCRIPTION
**Pull request by sriamar**

### CODE REVIEW

File: app/src/main/res/layout/activity_app_settings.xml [hunk 1]
```diff
@@ -84,6 +84,41 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_vertical"
+                android:text="@string/settings_title_presence"
+                android:textAppearance="?android:attr/textAppearanceLarge"/>
+        </FrameLayout>
+
+        <LinearLayout
```
Comment (smpete) [inline app/src/main/res/layout/activity_app_settings.xml]: We shouldn't need this encapsulating layout

File: app/src/main/res/layout/activity_app_settings.xml [hunk 2]
```diff
@@ -84,6 +84,41 @@
...
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"/>
+        </LinearLayout>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@color/atlas_view_divider"/>
+
+        <FrameLayout
```
Comment (smpete) [inline app/src/main/res/layout/activity_app_settings.xml]: I don't understand why we need these FrameLayouts. We should be able to get the same effect by adding margins to the TextView

File: build.gradle [hunk 3]
```diff
@@ -10,6 +10,7 @@ buildscript {
 
 allprojects {
     repositories {
+        maven { url "/Users/amar/.gitRepos/layerhq/releases-android/releases/" }
```
Comment (smpete) [inline build.gradle]: Remove local repo

Comment (sriamar) [inline build.gradle]: Done.

File: app/src/main/res/layout/activity_app_settings.xml (hunk 1 above)
Comment (sriamar) [inline app/src/main/res/layout/activity_app_settings.xml]: I thought so too. But, this whole page is designed that way. So, I followed the pattern as well. Thought about updating all this, and also about doing it differently for this. But, it didn't make sense to invest more time there. So, I followed the existing pattern.

File: app/src/main/res/layout/activity_app_settings.xml (hunk 2 above)
Comment (sriamar) [inline app/src/main/res/layout/activity_app_settings.xml]: Same as above.

**Pull request merged**